scripts/linear-ticket/validate.py

```
@@ -0,0 +1,459 @@
+            data = self.gh.get(f"/repos/{self.gh.repo}/pulls/{number}")
+            if data and data.get("state") == "open":
+                open_prs.append(number)
+
+        if len(open_prs) != 1:
+            error(f"Expected exactly one open PR associated with {head_sha}, found "
+                  f"{len(open_prs)} (event={wr.get('event')}). Refusing to publish an "
+                  "ambiguous result.")
+            return None
+        return open_prs[0]
+
+    def _query_attachments(self, html_url: str):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_query_attachments`

(ANN202)